samples/vms_api.js

```diff
@@ -22,71 +22,51 @@ const compute = google.compute('v1');
 // [END initialize]
 
 // [START auth]
-function auth(callback) {
-  google.auth.getApplicationDefault(function(err, authClient) {
-    if (err) {
-      return callback(err);
-    }
+async function auth() {
```

Comment:
This whole thing can be shorted to:

```js
return google.auth.getClient({
  scopes: [
      'https://www.googleapis.com/auth/cloud-platform',
      'https://www.googleapis.com/auth/compute',
      'https://www.googleapis.com/auth/compute.readonly'
    ]
});
```

Reply:
Actually, can you just file an issue to come back around and fix this part later?

Reply:
Ya sure. i will file this issue.